docs/resources/portal_auth.md

```diff
@@ -45,20 +45,20 @@ resource "konnect_portal_auth" "my_portalauth" {
 ### Optional
 
 - `basic_auth_enabled` (Boolean) The organization has basic auth enabled.
-- `idp_mapping_enabled` (Boolean) Whether IdP groups determine the Konnect Portal teams a developer has. This will soon replace oidc_team_mapping_enabled.
+- `idp_mapping_enabled` (Boolean) Whether IdP groups determine the Konnect Portal teams a developer has.
 - `konnect_mapping_enabled` (Boolean) Whether a Konnect Identity Admin assigns teams to a developer.
-- `oidc_auth_enabled` (Boolean) The organization has OIDC disabled.
-- `oidc_claim_mappings` (Attributes) Mappings from a portal developer atribute to an Identity Provider claim. (see [below for nested schema](#nestedatt--oidc_claim_mappings))
-- `oidc_client_id` (String)
-- `oidc_client_secret` (String)
-- `oidc_issuer` (String)
-- `oidc_scopes` (List of String)
-- `oidc_team_mapping_enabled` (Boolean) Whether IdP groups determine the Konnect Portal teams a developer has.
-- `saml_auth_enabled` (Boolean) The portal has SAML enabled or disabled.
+- `oidc_auth_enabled` (Boolean, Deprecated) Deprecated. Use the [Identity Provider API](https://developer.konghq.com/api/konnect/portal-management/v3/#/operations/update-portal-identity-provider) instead.
```

Comment:
Since portal identity provider is suggested, I looked into the endpoints, and seems they are still marked `x-unstable`
Asked about GA here https://kongstrong.slack.com/archives/CAW1MAFLZ/p1764582811733329